src/lib.rs

```diff
@@ -40,6 +40,40 @@ pub(crate) trait CastPtr {
     }
 }
 
+#[macro_export]
+macro_rules! try_slice {
+    ( $ptr:expr, $count:expr ) => {
+        if $ptr.is_null() {
+            return crate::panic::NullParameterOrDefault::value();
+        } else {
+            unsafe { slice::from_raw_parts($ptr, $count as usize) }
+        }
+    };
+}
+
+#[macro_export]
+macro_rules! try_mut_slice {
+    ( $ptr:expr, $count:expr ) => {
+        if $ptr.is_null() {
+            return crate::panic::NullParameterOrDefault::value();
+        } else {
+            unsafe { slice::from_raw_parts_mut($ptr, $count as usize) }
+        }
+    };
+}
+
+#[macro_export]
+macro_rules! ffi_panic_boundary_generic {
```

Comment:
I don't think this macro is used anywhere. Shouldn't everything just use `panic::ffi_panic_boundary`?